src/home/hd-home.c

```diff
@@ -1062,13 +1062,6 @@ hd_home_desktop_key_release (XKeyEvent *xev, void *userdata)
       (priv->key_sent == KEY_SENT_NONE) && (!STATE_IS_TASK_NAV(hd_render_manager_get_state())))
       return;
 
-  if((XkbKeycodeToKeysym(clutter_x11_get_default_display(), xev->keycode, 0, 0) == GDK_Control_L) &&
```

Comment:
Where is this block in hd-shortcuts.c? What it was replaced with?

Reply:
it was removed, action left button dose the same thing, and can be assigned to whatever key combination you want in shortcuts.ini